src/main/scala/com/github/t3hnar/bcrypt/package.scala

```diff
@@ -10,11 +12,11 @@ package object bcrypt {
   implicit class Password(val pswrd: String) extends AnyVal {
     def bcrypt: String = B.hashpw(pswrd, BCrypt.gensalt())
 
-    def bcrypt(rounds: Int): String = B.hashpw(pswrd, BCrypt.gensalt(rounds))
+    def bcrypt(rounds: Int): Try[String] = Try(B.hashpw(pswrd, BCrypt.gensalt(rounds)))
```

Comment:
I can definitely do that 👍 
So would you like me to suffix all the new APIs with `Safe`?